packages/rspack/src/executors/rspack/schema.d.ts

```diff
@@ -16,13 +18,20 @@ export interface RspackExecutorSchema {
 
   rspackConfig: string;
   optimization?: boolean | OptimizationOptions;
-  sourceMap?: boolean | string;
+  sourceMap?: boolean | 'hidden';
```

Comment:
hmmm I think this should be a string to not restrict a type from being passed in.
For example `inline-source-map` cannot be added which is useful in dev environments. 
The webpack one needs to be updated to match this.

Reply:
Lets create a task to update the webpack one that we can circle back to